cozy/cozy-flagship-app
# fix: force light theme on webviews if needed

# Add Color Scheme Meta Tag Functionality

## Summary
This PR introduces functionality to check for the existence of a `color-scheme` meta tag in an HTML string, and add it if it's not present. This ensures that the pages have the 'light only' theme applied if they do not explicitly specify a theme via the `color-scheme` meta tag.

## Changes
- Add `addColorSchemeMetaIfNecessary` function to check for the presence of a `color-scheme` meta tag and add it with 'light only' content if not present.
- Add tests for `addColorSchemeMetaIfNecessary` to ensure correct functionality.

## Testing
- Unit tests have been added to verify that the `color-scheme` meta tag is added when not present.
- Unit tests verify that if the `color-scheme` meta tag is already present, it is not added again.

## How to review
- Check that the code changes meet the coding standards and best practices.
- Ensure the tests cover all cases.
- Check for any edge cases or scenarios that may not have been considered.

## Note
This change is particularly important for ensuring that web pages rendered with our system adhere to the desired theming conventions.

## Preview
```javascript
export const addColorSchemeMetaIfNecessary = (HTMLstring: string): string => {
    // ...
};

// Test
describe('HTML Meta Attributes', () => {
    // ...
});
